### PR TITLE
.DS_Store is usually from macOS & .idea/ is IntelliJ products

### PR DESCRIPTION
**Pull request by xSavitar**

### CODE REVIEW

File: .gitignore [hunk 1]
```diff
@@ -3,6 +3,8 @@
 .vs/
 .vscode/
 build/
+.idea/
+.DS_Store
```
Comment (ThakeeNathees) [inline .gitignore]: Could you move these 2 just above the `build/`. To make it ordered by the hidden files `.foo` comes first

**Review by ThakeeNathees — CHANGES_REQUESTED** [review]

**Comment by ThakeeNathees** [comment]
It's a good idea to ignore project files ourselves, instead of expecting them to add it to their [local user .gitignore](https://git-scm.com/book/en/v2/Customizing-Git-Git-Configuration#_core_excludesfile). I'd just want a tiny change in the review.

File: .gitignore (hunk 1 above)
Comment (xSavitar) [reply .gitignore]: Sure will do that.

Comment (xSavitar) [reply .gitignore]: Done!

**Review by ThakeeNathees — APPROVED** [review]

**Pull request merged**